ui/src/app/components/direct-connection/direct-connection.component.scss

```diff
@@ -14,4 +14,7 @@
   button {
     margin-right: 20px;
   }
+  .form-container {
```

Comment:
The suggestion was to remove white background but its still there : https://drive.google.com/file/d/1vtJYbfoWzxmhQSf_ED15mmo58-6SCsRb/view?usp=sharing&resourcekey=0-uQ_BNiUOwqYldZeBgwrmfQ

Reply:
Removed white background and moved the common css of direct-connection, load-dump and load-session components to global style sheet.